### ISSUE TITLE
TensorFlow实现Softmax Regression识别手写数字 | asdfv1929 's Home

### ISSUE THREAD

**Issue by asdfv1929** [asdfv1929/asdfv1929.github.io#6]
https://asdfv1929.github.io/2018/01/05/tf-softmax-mnist/ 

 这是本人在学习TensorFlow实战一书时所记录下来的一些内容。
